scripts/check_benchmark_regression.py

```
@@ -20,6 +20,14 @@
         # each round. OS page-cache state on shared runners causes 3–5x variation between
         # consecutive CI runs, making this ungatable with any reasonable slack.
         "test_summary_cache_round_trip",
+        # Sub-100µs in-memory cache lookups vary 2.5x+ between consecutive ubuntu-latest
+        # runs; gated ratio band (0.5x–1.2x) cannot bracket both without false failures.
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)